mobile/src/actions/mediaBackup.js

```diff
@@ -33,8 +45,11 @@ export const mediaBackup = (dir) => async (dispatch, getState) => {
     let photos = await getFilteredPhotos()
     const alreadyUploaded = getState().mobile.mediaBackup.uploaded
     const dirID = await getDirID(dir)
+    const totalUpload = photos.length - alreadyUploaded.length
```

Comment:
That is a very good remark. Please find https://github.com/cozy/cozy-files-v3/pull/184/commits/77a6a491cd71403a6125573ea5b5d93669a7e69b to fix this.